### PR TITLE
Adds two new traits: Missing Right Eye and Missing Left Eye.

### PR DESCRIPTION
**Pull request by Juggernaut2442**
## About the PR 
Adds two new traits: 
- Missing Right Eye
image

- Missing Left Eye
image

Both cost zero points and just remove your eye of choice.

## Why's this needed? 
More customization options for the people that like to feel cool or just hate enjoying the game. 

## Changelog

```changelog
(u)Juggernaut2442
(*)Adds two new traits; Missing Right Eye and Missing Left Eye.
```

### CODE REVIEW

**Comment by flrsh** [comment]
This feels like trait bloat. Not every bit of customization should be obtainable through traits / at roundstart.

File: code/procs/jobprocs.dm [hunk 1]
```diff
@@ -682,6 +682,18 @@
...
+			SPAWN(6 SECONDS)
+				qdel(src.get_organ("left_eye"))
+				boutput(src, "<b>You feel the world around you lose its depth.</b>")
+
+	if (src.traitHolder && src.traitHolder.hasTrait("missingeyeR"))
+		if (src.organHolder.right_eye)
+			SPAWN(6 SECONDS)
+				qdel(src.get_organ("right_eye"))
+				boutput(src, "<b>You feel the world around you lose its depth.</b>")
+
```
Comment (Zonespace27) [inline code/procs/jobprocs.dm:696]:
```suggestion
	if (traitHolder?.hasTrait("missingeyeL"))
		if (organHolder.left_eye)
			SPAWN(6 SECONDS)
				qdel(get_organ("left_eye"))
				boutput(src, "<b>You feel the world around you lose its depth.</b>")

	if (traitHolder?.hasTrait("missingeyeR"))
		if (organHolder.right_eye)
			SPAWN(6 SECONDS)
				qdel(get_organ("right_eye"))
				boutput(src, "<b>You feel the world around you lose its depth.</b>")

```

**Comment by Kyle2143** [comment]
missing a single eye is really terrible rn, not at all like what having only one working eye should be like I'm going to change it to be less oppressive.

**Comment by Juggernaut2442** [comment]
Ye, single eye is really bad right now. Not only because it blocks half your screen but the MEATY overlay makes the visible side darker than it should be. My suggestions was removing the overlay completely and adding an accuracy debuff to weapons/thrown items and a chance to drop/break items that are being placed on a table and such. Or at least temporarily removing the MEATY overlay since fixing it seems to be quite the hassle.

**Comment by Juggernaut2442** [comment]
> This feels like trait bloat. Not every bit of customization should be obtainable through traits / at roundstart.

Gotcha! Will close soon then. 👍

**Pull request closed without merging**